diff --git a/cmake/HuxerUIRuntimeDependencies.cmake b/cmake/HuxerUIRuntimeDependencies.cmake
--- a/cmake/HuxerUIRuntimeDependencies.cmake
+++ b/cmake/HuxerUIRuntimeDependencies.cmake
@@ -673,7 +673,8 @@ function(_huxerui_deploy_runtime source_executable content_directory executable)
         find_program(patchelf patchelf REQUIRED)
         find_program(readelf readelf REQUIRED)
         set(library_directory "${content_directory}/lib")
         set(system_excludes "^ld-linux.*" "^lib(c|m|dl|pthread|rt|resolv|util|anl|BrokenLocale)\\.so(\\.[0-9]+)*$"
-                "^libstdc\\+\\+\\.so\\.6$" "^libgcc_s\\.so\\.1$")
+                "^libstdc\\+\\+\\.so\\.6$" "^libgcc_s\\.so\\.1$"
+                "^lib(ssl|crypto)\\.so(\\.[0-9]+)*$")
         set(desktop_runtime_regex "^lib(gtk-4|epoxy|gio-2\\.0|glib-2\\.0|gobject-2\\.0|pango-1\\.0|cairo|soup-3\\.0|EGL|GL|GLX|GLdispatch|OpenGL|GLESv[12]|vulkan|drm)\\.so(\\.|$)")
     endif ()
